src/main/scala/tech/cryptonomic/conseil/tezos/PlatformDiscoveryOperations.scala

```diff
@@ -88,4 +93,72 @@ object PlatformDiscoveryOperations {
   private def makeDisplayName(name: String): String = {
     name.capitalize.replace("_", " ")
   }
+
+
+  /** Makes list of possible string values of the attributes
+    * @param  tableName name of the table from which we extract attributes
+    * @param  attribute name of the attribute
+    * @param  withFilter optional parameter which can filter attributes
+    * @return list of attributes
+    **/
+  def listAttributeValues(tableName: String, attribute: String, withFilter: Option[String] = None)
+    (implicit ec: ExecutionContext): Future[List[String]] = {
+    val res = verifyAttributesAndGetQueries(tableName, attribute, withFilter)
+    ApiOperations.runQuerySequence(res).map(_.flatten)
+  }
+
+  /** Makes list of DBIO actions to get possible string values of the attributes
+    * @param  tableName name of the table from which we extract attributes
+    * @param  attribute name of the attribute
+    * @param  withFilter optional parameter which can filter attributes
+    * @return list of DBIO actions to get matching attributes
+    **/
+  def verifyAttributesAndGetQueries(tableName: String, attribute: String, withFilter: Option[String])
```

Comment:
Is it feasible here to return a `DBIO[List[String]]` by `DBIO.sequence`-ing the `for` result and then `.map(_.flatten)` the ensuing `DBIO[List[List[String]]`?

For me it would make more sense to present a simpler type to the caller, instead of forcing him to flatten himself... unless the single operations in the list are to be used individually somewhere else...

Reply:
ok, I'll change it but my reasoning was that I didn't want to repeat DBIO.sequence

Reply:
I thought about that too, but personally I don't think this should be a driving factor behind code design choices. I tend to focus more on the intent and ergonomics of the resulting interface.